debian/nfd.service

```diff
@@ -32,7 +32,10 @@ After=nfd-keygen.service
 [Service]
 EnvironmentFile=-/etc/default/nfd
 Environment=HOME=/var/lib/ndn
-ExecStart=/bin/sh -c 'LANG=C /usr/bin/nfd --config /etc/ndn/nfd.conf ${OPTIONS}'
+# FIXME: Needed to work around libboost bug
+# https://svn.boost.org/trac/boost/ticket/4688
+Environment=LANG=C
+ExecStart=/usr/bin/nfd --config /etc/ndn/nfd.conf ${OPTIONS}
```

Comment:
Where is the non-`C` `LANG` set in your environment? Do you have any options set in `etc/default/nfd` which could interfere here? You could debug what the environment is when `nfd` crashes by temporarily changing `ExecStart` to
```
ExecStart=/bin/sh -c 'env > /var/lib/ndn/env.log; /usr/bin/nfd --config /etc/ndn/nfd.conf ${OPTIONS}'
```

I get the following:
```
HOME=/var/lib/ndn
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
LANG=C
PWD=/
DAEMON_OPTS=
```